README.md

```diff
@@ -1,18 +1,51 @@
 # Polling App
 
+ [![contributions welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat)](https://github.com/sarthakkundra/polling-app)
+
+ ![GitHub contributors](https://img.shields.io/github/contributors/sarthakkundra/polling-app) ![GitHub issues](https://img.shields.io/github/issues-raw/sarthakkundra/polling-app?color=sucess) ![GitHub pull requests](https://img.shields.io/github/issues-pr/sarthakkundra/polling-app?color=blueviolet)
+
+----
+        
 ## About the Project
 This is a web app similar to Kahoot. Users can post questions with multiple choices and other users can vote on it. The result of each question can be viewed in real-time in the form of charts.
 
 ## Tech Stack
-1) Node.js
-2) React.js
-3) MongoDB
-4) Ant Design
-
-### Getting Started
-1) Fork the repo.
-2) Clone the repo.
-3) Run npm install in the root directory.
-4) Run npm install in 'client' directory.
-5) Run npm run dev in the root directory
-6) App is up on port 3000.
+* [Node.js](https://nodejs.org/en/docs/)
+* [React.js](hhttps://reactjs.org/docs/getting-started.html)
+* [MongoDB](https://docs.mongodb.com/guides/)
+* [Ant Design](https://ant.design/docs/react/introduce)
+
+---
+
+## Getting Started with the project :
+### Step 1 : 
+#### Fork the repository
+<br>
+
+### Step 2 :
+#### Clone the repo. in your local computer by typing the following command in the command line
+` git clone https://github.com/sarthakkundra/polling-app.git `
+<br>
+
+### Step 3 :
+#### Run `npm install` in the root directory to install all the necessary dependencies
+<br>
+
+### Step 4 :
+#### Run `npm install` in [client](client) directory for setting up the React.js application.
+<br>
+
+### Step 5 :
+#### Run `npm run dev` in the root directory for starting the application.
+<br>
+
+### Step 6 :
+#### Your application is up and running on port `127.0.0.1:3000` or `localhost:3000`.
+
+----
```

Comment:
The project now requires environment variables to be setup before starting. So write a step about setting up env variables between step 4 and step 5. The variables are 

- GITHUB_TOKEN (optional) (can be generated from the Github profile)

- PORT (number)

-  JWT_SECRET (string)

A .env file needs to be created in the root directory with the following variables. Let me know if you have any doubts regarding this.


-

Reply:
@sarthakkundra sir how to generate the `GITHUB_TOKEN` and the `JWT_SECRET` and also shall we consider adding the **mongodb atlas** insted of local mongodb

Reply:
and also @sarthakkundra sir shall I have to make changes in the **same branch** at my fork and make a pull request again to be able to merged by you

Reply:
@sarthakkundra sir please do acknowledge this [issue](https://github.com/sarthakkundra/polling-app/issues/1#issuecomment-696398681)

Reply:
You can refer to this [https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/creating-a-personal-access-token](link) for the github token and JWT_SECRET can by any string.